extra/pom.xml

```diff
@@ -120,6 +121,11 @@
                 <artifactId>commons-compress</artifactId>
                 <version>${commons.compress.version}</version>
             </dependency>
+            <dependency>
+                 <groupId>commons-logging</groupId>
+                 <artifactId>commons-logging</artifactId>
+                 <version>${commons-logging.version}</version>
+            </dependency>
```

Comment:
Why are you adding this new dependency?

Reply:
It's getting pulled in at a vulnerable version by another dependency, and this will pin it at a non-vulnerable version.

Reply:
@JimTharioAmazon I think what @CTMBNara meant is that the dependency waws pulled into the project just to define the version. This is usually an extreme measure. There are 3 material project dependencies pulling in the `commons-logging` library: `httpclient`, `mockserver-client-java` and `google-cloud-storage` by one of the modules. The correct way forward is to look into the actual usage. I did a quick glance and you should be able to exclude this dependencies from all of them. And the `google-cloud-storage` dependency could also be bumped.